### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by Zakhar-Hadanovich** [DW8030/Civ6-mod#48]
Good day. I'm not sure, but I think I found a bug in the civic tree. The ilkum card (building) is not available for construction when you open craftsmanship. The civilopedia says that in order to unlock ilkum, you still need to open feudalism, but if I'm not mistaken, the ilkum card should become obsolete in feudalism, and not open. But logically, it turns out that the card is unavailable, and in feudalism it is generally written that ilkum, after its adoption, becomes obsolete. Or is there something I don't understand?

**Comment by DW8030** [comment]
No, you understand it. I had accidentaly typed it was available 'after' instead of 'before' adopting Feudalism, in which case you have to adopt both Craftmanship and Feudalism. Thank you for spotting the error, I fixed it.
